hatchways/team-cosmic-crisp
# Fe refactor payment

### What this PR does (required):
- refactor payment

### Screenshots / Videos (required):

https://user-images.githubusercontent.com/6465981/121000830-84370f00-c73f-11eb-8b7a-55423ff120f6.mp4



### Any information needed to test this feature (required):
- use sitter@email.com, 123456 for dog sitter,
- use owner@email.com, 123456 for dog owner
- go to listing page, find Sitter Dog, and create a new request
- on sitter's panel, go to request page, the new request should be there
- click the vertical three dot to accept/decline the request
- if accept, owner is able to pay for the dog sitting.

### Any issues with the current functionality (optional):
